Kitodo-DataManagement/src/main/java/org/kitodo/data/database/persistence/apache/StepManager.java

```diff
@@ -18,11 +18,12 @@
 import java.util.List;
 import java.util.Map;
 
-import org.apache.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class StepManager {
 
-    private static final Logger logger = Logger.getLogger(MySQLHelper.class);
+    private static final Logger logger = LogManager.getLogger(MySQLHelper.class);
```

Comment:
Can you please correct the used class name from `MySQLHelper.class` to `StepManager.class` as the class name is `StepManager`?